docs/data-sources/aviatrix_all_network_domains.md

```diff
@@ -0,0 +1,38 @@
+---
+subcategory: "Multi-Cloud Transit"
```

Comment:
Network Domains are under `TGW Orchestrator`, so this should probably be under that subcategory too.